articles/user-guide/index.md

```diff
@@ -13,34 +13,16 @@ no-loc: ['Q#', '$$v']
 
 Welcome to the Q# User Guide! 
 
-In the different topics of this guide, we detail the core concepts of the Q# language and all the information you need to write quantum programs.
+In the different topics of this guide, we introduce some of the basics for developing quantum programs using Q#.
+
+We refer to the [Q# GitHub repository](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language#q-language) for a full specification and documentation of the Q# quantum programming language. 
 
 ## User Guide Contents
 
-- [Q# Basics](xref:microsoft.quantum.guide.basics): An introductory overview of the purpose and functionality of the Q# programming language. 
+- [Q# programs](xref:microsoft.quantum.guide.programs): An quick introduction to quantum programs in Q#. 
 
 - [Ways to run a Q# program](xref:microsoft.quantum.guide.host-programs): describes how a Q# program is run, and provides an overview of the various ways you can call the program: from the command line, in Q# Jupyter Notebooks, or from a classical host program written in Python or a .NET language.
 
```

Comment:
The link above to the Q# GitHub repository points to a page that has such an index. We could certainly duplicate it here. I wasn't sure if it would make more sense to do that when the pages are on the docs repo?

Reply:
@Bradben Given that the PR #1075 introduces that index, I suggest to merge this PR as is.